lib/rules/disallow-space-after-keywords.js

```diff
@@ -51,7 +51,9 @@ module.exports.prototype = {
         );
 
         if (keywords === true) {
-            keywords = defaultKeywords;
+            keywords = defaultKeywords.filter(function(keyword) {
+                return ['function', 'return'].indexOf(keyword) === -1;
```

Comment:
Although actually - maybe it shouldn't check `function` since we have different rules to check function spacing - and other stuff like `function *` which should be a rule